### PR TITLE
Make title card at all pages to be consistent

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Description

add header at all pages to be consistent 

### Changes

all these pages need a title card
Miniting
Node Finder
VM https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2908
orchestrators https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2908
Application
Contracts
SSH Key
Farm Finder

### Related Issues

#2908
#2224

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

image
image
image
image
image
image
image
image

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/views/nodes.vue [hunk 1]
```diff
@@ -1,11 +1,20 @@
 <template>
-  <div class="hint">
+  <!-- <div class="hint">
     <v-alert class="mb-4" type="info" variant="tonal">
       Node status is updated every 90 minutes. For a realtime status, click on the node's card.
     </v-alert>
-  </div>
+  </div> -->
```
Comment (Mahmoud-Emad) [inline packages/playground/src/views/nodes.vue:6]: Do we really need to comment this out? I noticed that you moved it below, so let's delete it.

File: packages/playground/src/views/sshkey_view.vue [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <template>
-  <v-card class="pa-6 mb-4">
+  <view-layout>
+    <v-card color="primary" class="d-flex justify-center items-center pa-3 mb-3 text-center">
+      <v-icon size="30" class="pr-3">mdi-key-plus</v-icon>
+      <v-card-title class="pa-0">SSH keys</v-card-title>
+    </v-card>
+    <!-- <v-card class="pa-6 mb-4">
     <div>
       <h3 class="text-light">
```
Comment (Mahmoud-Emad) [inline packages/playground/src/views/sshkey_view.vue:9]: Here also please.

**Review by Mahmoud-Emad — COMMENTED** [review]
Good job ya MR Ehab!

**Review by Mahmoud-Emad — APPROVED** [review]
Go ahead!

**Review by samaradel — APPROVED** [review]

**Pull request merged**